### DISCUSSION TITLE
Registration not working

### DISCUSSION THREAD

**Discussion by Raphael579** [keycloak/keycloak#23104] [Archive]
Hello,

I have a frontend application connected to Keycloak for user authentication. Currently, users can log in successfully, but I'd like to enable user self-registration for account creation. I've watched several tutorials and it seems all I need to do is enable user registration in Keycloak. I've done that, but now when I click on "Register" in the browser to create a new account, Keycloak returns an error message saying, "Invalid username or password." I've tried searching online for a solution, but I couldn't find any relevant information. It appears to be related to a setting that might be blocking registrations.

Thank you in advance.

**Comment by ahmedsmaali5** [comment]
Hello there, 

Can you please share more information on your setup?

- Version?
- Installation method?
- Is this a fresh installation of keycloak?
- Do you experience same behavior in private browsing session?
- Does this happen with a newly created realm?
